src/main/telemetry/crsf.c

```diff
@@ -446,15 +446,17 @@ void initCrsfTelemetry(void)
 #endif
 
     int index = 0;
-    if (sensors(SENSOR_ACC)) {
+    if (sensors(SENSOR_ACC) && telemetryIsSensorEnabled(SENSOR_PITCH | SENSOR_ROLL | SENSOR_HEADING)) {
```

Comment:
`telemetryIsSensorEnabled()` needs to be stubbed out in the crsf tests as they are breaking (try `make test`).